fairseq/training.py

```diff
@@ -71,15 +132,14 @@ def __init__(
         fairseq_hydra_config,
         *,  # args below are keyword only
         command_line_args=None,
-        max_epoch=None,
-        save_interval=None,
+        max_epoch=1,
+        save_interval=1,
         keep_epochs=None,
         time_rqmt=4,
         mem_rqmt=4,
         cpu_rqmt=2,
         gpu_rqmt=1,
         fairseq_python_exe=None,
-        fairseq_hydra_exe=None,
         fairseq_root=None,
```

Comment:
below you assert that `fairseq_root is not None`, but here the default is None. Do you suggest that there should be no default then? In this case just remove the default. (Because of the `*,` above it should be allowed to not give any default here without reordering the arguments.
```suggestion
        fairseq_root,
```